NEW_PROJECT_APPLICATION.md

```diff
@@ -31,6 +31,8 @@ For each repository, please provide:
 * Who are the members of the leadership team?
 * How are members of the leadership team nominated?
 * How are individuals outside of leadership given commit access?
+* Is the project open/willing to move to more open goverance (if not already in place)?
```

Comment:
I find this phrase "more open governance" to be ambiguous. I suggest we remove the "more" and it is more direct, especially with the parenthetical statement that follows.

Reply:
I would also suggest that we drop the parenthetical and move that to a top level assertion:

```suggestion
* Is the project open/willing to move to open governance, if not already in place?
```

Also, there's a tiny typo fixed in the suggestion: `goverance` => `governance`